### ISSUE TITLE
[Feature Request] Add [Ctrl] + [+/-] and [Ctrl] + [Mouse Scroll] for change UI zoom level and save it for each monitor

### ISSUE THREAD

**Issue by hmdhasani** [telegramdesktop/tdesktop#16986]
### Is your feature request related to a problem?

If you are using multiple screen with different resolutions and size (pixel density) moving telegram between them will cause in very large/small UI that need to frequently change the interface scale.

### Describe the solution you'd like

1) Add a common shortcuts for zoom that are:
    - [Ctrl] + [+/-]
    - [Ctrl] + [Mouse Scroll]
 
2) Save the scale level based on each monitor so if the telegram moved between them the zoom level automatedly change

### Describe alternatives you've considered

currently I use scale change from settings.

### Additional context

Sample for current UI on two monitors:

1) 1920*1080 - 23 Inches
![image](https://user-images.githubusercontent.com/11094468/134153486-d04f6dfc-dd42-4aca-9a4a-ff30a106104f.png)

2) 2736*1824 - 12 Inches
![image](https://user-images.githubusercontent.com/11094468/134153736-3827bc8c-bc16-479e-8d99-ed3d80bc4a9b.png)

**Comment by ilya-fedin** [comment]
This request has no sense since restart is required to change interface scale. And if you can change interface scale in runtime, it's better to implement automatic scale change when moving between monitor. So I think this FR should be closed in favor of #1121.

**Comment by hmdhasani** [comment]
I think automatic change of the scale will help a lot but the shortcut is still handy. For example the user may prefer some adjustment over the auto changed scale.

**Comment by hmdhasani** [comment]
You are right about my second request, but for my first request, adding shortcuts, it's a very common feature.

**Comment by ilya-fedin** [comment]
Can you name at least one program that provide possibility to change _interface_ scale with Ctrl+[+/-] and save that setting _per-monitor_? I never seen one.

**Comment by ilya-fedin** [comment]
Oh, it's the second item

**Comment by LinFor** [comment]
> Can you name at least one program that provide possibility to change _interface_ scale with Ctrl+[+/-] and save that setting _per-monitor_? I never seen one.

Just for example VS Code changes interface scale with Ctrl+[+/-]. Seems it does not save it per-monitor, but multiply this user-specified interface scale by monitor's scale in real-time.
